src/main/java/spireMapOverhaul/zones/volatileGrounds/powers/ChargedPower.java

```diff
@@ -0,0 +1,58 @@
+package spireMapOverhaul.zones.volatileGrounds.powers;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.common.SuicideAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.combat.ExplosionSmallEffect;
+import spireMapOverhaul.abstracts.AbstractSMOPower;
+import spireMapOverhaul.zones.manasurge.ManaSurgeZone;
+import spireMapOverhaul.zones.volatileGrounds.VolatileGrounds;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+
+public class ChargedPower extends AbstractSMOPower {
+    public static final String ID = makeID("volChargedPower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+    public static String ZONE_ID = VolatileGrounds.ID;
+    
+    public ChargedPower(AbstractCreature owner, int amount) {
+        super(ID, NAME, ZONE_ID, PowerType.BUFF, false, owner, amount);
+        updateDescription();
+    }
+    @Override
+    public void updateDescription() {
+        this.description = DESCRIPTIONS[0].replace("{0}", this.amount + "");
+    }
+    
+    @Override
+    public void onDeath() {
+        this.addToBot(new VFXAction(new ExplosionSmallEffect(this.owner.hb.cX, this.owner.hb.cY), 0.1F));
+        AbstractDungeon.actionManager.addToBottom(new AbstractGameAction() {
+            @Override
+            public void update() {
+                AbstractDungeon.player.damage(new DamageInfo(owner, ChargedPower.this.amount, DamageInfo.DamageType.THORNS));
+                for (AbstractMonster target : AbstractDungeon.getCurrRoom().monsters.monsters) {
+                    target.tint.color.set(Color.RED);
```

Comment:
Will this mess with the color Red?

Reply:
I doubt it? The basegame uses it the same way for damaging actions and I did not see it cause any issues.

Reply:
Discussed, this is safe and won't mess with `Color.RED`.